account_brand/models/account_invoice.py

```diff
@@ -16,3 +16,10 @@ class AccountInvoice(models.Model):
             'cancel': [('readonly', True)],
         }
     )
+
+    @api.constrains('brand_id', 'company_id')
```

Comment:
This `@api.constrains` should depend also on `type` IMO.
And `'company_id'` should be `'company_id.brand_use_level'` instead, but this should be also fixed in the base `brand` module.

Reply:
Indeed I agree, I'll proceed to the change.